fsharp-backend/src/BackendOnlyStdLib/HttpClient.fs

```diff
@@ -319,7 +319,7 @@ let rec httpCall
 
           // CLEANUP no reason to do this
           // Match curls default redirect behaviour: if it's a POST with content, redirect to GET
-          // FSTODO: are some headers involved
+          // FSTODO: are some headers involved EXTRACT
```

Comment:
I'm going to extract this FSTODO and the one in `LegacyBaseHttpClient` into their own joined issue.

Reply:
Good idea